### PR TITLE
Fix adui 4535 connect react vapor input to selector

### PR DESCRIPTION
**Pull request by dmgauthier**
### Proposed Changes

then again trying to do just right by using the components selectors

### Potential Breaking Changes

none

### Acceptance Criteria

-   [ ] The proposed changes are covered by unit tests
-   [ ] The potential breaking changes are clearly identified
-   [ ] [README.md](https://github.com/coveo/react-vapor/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

### CODE REVIEW

File: packages/react-vapor/src/components/radio/RadioSelectSelectors.ts [hunk 1]
```diff
@@ -13,7 +13,12 @@ const getValue = createSelector(
     (radioSelect: IRadioSelectState) => radioSelect && radioSelect.value
 );
 
+const getDisabledValue = createSelector(
```
Comment (gdostie) [inline packages/react-vapor/src/components/radio/RadioSelectSelectors.ts]: Unit tests for this new selector?

Comment (dmgauthier) [reply packages/react-vapor/src/components/radio/RadioSelectSelectors.ts]: - [ ] Unit Test for getDisabledValue Selector

File: packages/react-vapor/src/components/radio/tests/RadioSelectSelectors.spec.ts [hunk 2]
```diff
@@ -41,4 +44,27 @@ describe('RadioSelectSelectors', () => {
             expect(selectedValue).toBe(expectedRadioSelect.value);
         });
     });
+
+    describe('getdisabledValue', () => {
```
Comment (GermainBergeron) [inline packages/react-vapor/src/components/radio/tests/RadioSelectSelectors.spec.ts]:
```suggestion
    describe('getDisabledValue', () => {
```

Comment (dmgauthier) [reply packages/react-vapor/src/components/radio/tests/RadioSelectSelectors.spec.ts]: :open_mouth: thanks!

**Pull request merged**